lib/api/v1/controllers/authentication.go

```diff
@@ -22,25 +25,35 @@ func (c *AuthenticationController) Login(ctx *gin.Context) {
 
 	err := ctx.ShouldBindWith(authenticationForm, binding.Form)
 	if err != nil {
-		ResponseWithError(ctx, http.StatusBadRequest, err)
+		ResponseWithError(ctx, http.StatusBadRequest, err, constants.ERROR_CODE_MALFORM_REQUEST)
```

Comment:
@carryall I think this is a challenging one by grouping `status` and `code` into one.
So the benefit of this is a one-stop place for error managing 🖖
The idea from here https://github.com/go-oauth2/oauth2/blob/master/errors/response.go

Reply:
Thanks, @Lahphim  I'll work on it in [a separate issue](https://github.com/carryall/go-google-scraper-challenge/issues/94)